### ISSUE TITLE
Make the Atkinson Hyperlegible Font available for books

### ISSUE THREAD

**Issue by MichSpieler** [pressbooks/pressbooks-book#1344]
A network manager has requested to make the [Atkinson Hyperlegible](https://www.brailleinstitute.org/freefont/) Font available for books for accessibility reasons.

The font is available at https://fonts.google.com/specimen/Atkinson+Hyperlegible, but out of data privacy concerns they don't want to include the Google hosted version via CSS (referring specifically to this page: https://www.cookieyes.com/documentation/google-fonts-and-gdpr/ which declares that according to a German court, Google Fonts violates the GDPR).

([Link](https://pressbooks.zendesk.com/agent/tickets/26143) to request (for PB staff only))

**Comment by SteelWagstaff** [comment]
@cmurtagh do you mind if I take this one? I'm happy to add this font as an option to our 'Shapeshifter' feature.

**Comment by SteelWagstaff** [comment]
I opened two PRs that add support for the Atkinson Hyperlegible Next font recommended by the creators:
> We recommend installing Atkinson Hyperlegible Next to start—it’s our latest and greatest, and suitable for everyday readers.

**Comment by richard015ar** [comment]
I tested and reviewed these PRs. I pushed a friendly commit in one of them for a linting issue in an unchanged file, likely due to a previous change in the dev branch.

**Comment by tw77** [comment]
Are we sure this is on dev networks? In a dev.pressbooks.network book I don't see Atkinson Hyperlegible Next as one of the available header or body fonts in Theme Options, regardless of selected book theme.

**Comment by tw77** [comment]
Deployed to dev late last Thursday, works as expected, ready for release

**Comment by tw77** [comment]
This has been released to production networks
